module/programs/helper.js

```diff
@@ -976,6 +977,153 @@ module.exports = class ProgramsHelper {
     });
   } 
 
+  /**
+  * Program join.
+  * @method
+  * @name join
+  * @param {String} programId - Program Id.
+  * @param {Object} data - body data.
+  * @param {String} userId - Logged in user id.
+  * @param {String} userToken - User token.
+  * @param {String} [appName = ""] - App Name.
+  * @param {String} [appVersion = ""] - App Version.
+  * @returns {Object} - Details of the program join.
+  */
+
+  static join( programId, data, userId, userToken, appName = "", appVersion = "", internalAccessToken = "" ) {
```

Comment:
@VISHNUDAS-tunerlabs let's not pass the token itself here, we can use a flag called callConsetAPIOnBehalfOfUser